lib/report-builder-factory/report-builder.js

```diff
@@ -104,8 +104,8 @@ module.exports = class ReportBuilder {
         return this;
     }
 
-    setExtraItems(extraItems) {
-        this._extraItems = extraItems;
+    setApiValues(values) {
```

Comment:
Why is it called `setApiValues`? What API is this about?

Reply:
it's about extra items and meta info extenders. I will think about more clearer name of this method